neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -95,21 +100,36 @@ class NeptuneCallback(Callback):
         You need to have Keras or Tensorflow 2 installed on your computer to use this module.
     """
 
-    def __init__(self, run: Run, base_namespace: str = "training"):
+    def __init__(
+        self,
+        run: Run,
+        base_namespace: str = "training",
+        log_model_diagram: bool = False,
+    ):
         super().__init__()
 
+        self.log_model_diagram = log_model_diagram
```

Comment:
Should be internal property `self._log_model_diagram` and placed after parameters verification

Reply:
Done